apps/ocr-extraction/README.md

```diff
@@ -46,6 +46,7 @@ docker run \
            -e DB_PASS="password" \
            -e WF_EXTRACT_IMAGE_TEXT=true \
            -e WF_EXTRACT_PDF_TEXT=true \
+           -e WF_EXTRACT_EMBEDDINGS=true \
```

Comment:
shouldn't this be `WF_GENEREATE_EMBEDDINGS` ( and in the parameters below as well? )